samples/msal-node-samples/ExpressTestApp/package.json

```diff
@@ -21,7 +21,7 @@
     "typescript": "^4.0.5"
   },
   "dependencies": {
-    "@azure/msal-node": "^1.1.0",
+    "@azure/msal-node": "file:../../../lib/msal-node",
```

Comment:
Let's leave this versioned. You can add scripts to help going back and forth between local and published versions. Some of our other samples demonstrate this.